transforms/__tests__/codemod-missing-await-act.js

```diff
@@ -651,11 +671,19 @@ test("export newly async warns", async () => {
 			await domAct(scope)
 		}"
 	`);
-	await expect(console.warn.mock.calls).toEqual([
-		[expect.stringContaining("test.tsx: Export 'act' is now async.")],
-		[expect.stringContaining("test.tsx: Export 'unstable_act' is now async.")],
-		[expect.stringContaining("test.tsx: Default export is now async.")],
-	]);
```

Comment:
We warn from the CLI now. Warnings in the transform  will be interleaved with other logs making it harder to spot.